pkgs/top-level/linux-kernels.nix

```diff
@@ -251,6 +259,15 @@ in {
 
     linux_latest_libre = deblobKernel packageAliases.linux_latest.kernel;
 
+    linux_rust = pkgs.linux_latest.override {
+      structuredExtraConfig = with lib.kernel; {
+        RUST = yes;
+        GCC_PLUGINS = no;
+        SAMPLES = yes;
+        SAMPLES_RUST = yes;
+      };
+    };
```

Comment:
Can we just enable it unconditionally?  We're going to want to soon enough anyway, and I'd rather not add more kernel variants.

Reply:
Sure, added it to common-config.nix. Currently (6.3) only x86_64 is supported.